client/java/src/main/java/io/openlineage/client/dataset/namespaceResolver/HostListNamespaceResolverBuilder.java

```diff
@@ -0,0 +1,24 @@
+/*
+/* Copyright 2018-2024 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+package io.openlineage.client.dataset.namespaceResolver;
+
+public class HostListNamespaceResolverBuilder implements DatasetNamespaceResolverBuilder {
```

Comment:
Would be awesome to get proper documentation of those in OpenLineage/docs, but I'm sure you've already thought of that.